### ISSUE TITLE
Cutting arm travel is limited

### ISSUE THREAD

**Issue by WegnerDan** [IRTrail/G2E-Filametrix#12]
I have a hard time describing this, so I made some videos:

Without the G2E on top, the travel of the cutting arm is fine:

https://github.com/IRTrail/G2E-Filametrix/assets/6908247/3e7bb79e-3903-4199-9b67-89a52a9f2ef2

When the hotend is installed to the toolhead mount, G2E limits the travel of the cutting arm more than it needs to. 

https://github.com/IRTrail/G2E-Filametrix/assets/6908247/6f8d3ab7-c530-49bf-a3ca-5c4a46384f43

I think it makes sense to either extend the top of the cutting arm outward a couple mm (1)
or allow the cutting arm to travel outward a little more (2).

This would make cutting the blade to the correct length a little easier. 
![image](https://github.com/IRTrail/G2E-Filametrix/assets/6908247/a9d10463-8859-46c3-a992-3165573e282a)

What do you think?

**Comment by WegnerDan** [comment]
Some more information:

The blade in the video was shortened to 25.3 mm, because it was interfering with the filament.

![PXL_20240522_213237679](https://github.com/IRTrail/G2E-Filametrix/assets/6908247/68c378c4-5c85-46f0-98f4-f55acd46547c)

This is a blade cut to 26 mm:
![PXL_20240522_212524018](https://github.com/IRTrail/G2E-Filametrix/assets/6908247/2289f37c-5604-4c8d-b3bf-f61fdac38824)


![PXL_20240522_212622632](https://github.com/IRTrail/G2E-Filametrix/assets/6908247/fd13f732-db7e-43c7-82c7-75a1238cf43c)


As a test, I've filed down the area marked with 2 in the screenshot above and the filament now clears the 26mm blade

**Comment by IRTrail** [comment]
When I made this, I matched the angular deflection of the original version by Sorted. The gap in the space between the arm and the G2E bodies is about half a milimeter, iirc (been a monute).

The stop is set to match deflection as well.

There's also the "wide arm" which retains the original hook that stops against the ADXL screw if you like. It's fully interchangeable.

You're the first to mention this issue. I'll keep an eye out for more reports.
